Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/NewMeterWizard.tsx

```diff
@@ -316,6 +316,22 @@ export default function NewMeterWizard(props: {IsEngineer: boolean}) {
         localStorage.setItem('NewMeterWizard.CurrentStep', currentStep.toString())
     }
 
+    function meterDetails() {
+        setError([]);
+        setWarning([]);
```

Comment:
First issue is this will override all other errors and warnings?
If you really need to add an error that should be done in the individual step.